src/styles/main.scss

```diff
@@ -166,3 +169,23 @@ button.p-component.p-button {
 .p-multiselect-panel .p-multiselect-header {
   display: none;
 }
+
+// Under construction styles
+button.under-construction {
+  background-color: transparent !important;
+  border-color: var(--blue-400) !important;
```

Comment:
Are you sure about the blue? I feel like it looks like just a different button style a bit too much. Maybe yellow?
<img width="1329" alt="afbeelding" src="https://github.com/user-attachments/assets/3583ca22-b351-482a-9120-7078a37e4858">

Reply:
Sure, I like it with yellow, will add the change